web/client/libs/ajax.js

```diff
@@ -122,11 +122,17 @@ axios.interceptors.request.use(config => {
             const cannotUseCORS = corsDisabled.reduce((found, current) => found || uri.indexOf(current) === 0, false);
             if (!isCORS && (!autoDetectCORS || cannotUseCORS)) {
                 const parsedUri = urlUtil.parse(uri, true, true);
+                const params = Object.keys(config.params || {}).reduce((acc, key) => {
+                    if (config.params[key] !== undefined && config.params[key] !== null) {
```

Comment:
Here you can use the isNil function from lodash
`if ( !isNil(config.params[key]) ) {`

or you can change it to a shorter solution as explained [here](https://stackoverflow.com/questions/30812765/how-to-remove-undefined-and-null-values-from-an-object-using-lodash)
`var params = omitBy(config.params, isNil);`